src/firebolt_ingest/table_model.py

```diff
@@ -31,12 +31,14 @@ def parse_yaml(cls, yaml_obj: Any):
     "VARCHAR",
     "STRING",
     "DATE",
+    "PGDATE",
     "DATETIME",
-    "TIMESTAMP",
```

Comment:
Should we keep TIMESTAMP as well? It will be an alias to TIMESTAMPNTZ when the new types are enabled.

Reply:
Makes sense